examples/model_compress/experiment_result/analyze.py

```diff
@@ -0,0 +1,93 @@
+import argparse
```

Comment:
the folder name `experiment_result` can be changed to `comparison_of_pruners`

Reply:
> the folder name `experiment_result` can be changed to `comparison_of_pruners`

ok, thx